features/steps/cleaner_storage.py

```diff
@@ -149,6 +149,7 @@ def given_database_password(context, password):
 
 
 @given(u"database connection is established")
+@when(u"database connection is established")
```

Comment:
you can use `@step` instead of using `given` + `when`. `@step` will work for any kind of step